main.nf

```diff
@@ -275,6 +282,20 @@ process odgiDraw {
   """
 }
 
+process gzipOutputFiles {
```

Comment:
Would it make sense to rename this into `pigzOutputFiles`? To make it clearer, which tool the process is using.